app/scripts/controllers/tools.js

```diff
@@ -513,8 +526,11 @@ angular.module('oncokbApp')
                                                     mutation: getAlterations(item.alterations),
                                                     tumorType: tempTT,
                                                     tumorSummary: item.description,
+                                                    summary_uuid: item.uuid,
```

Comment:
Let's keep the same format when naming the property.

Reply:
Changed to summaryUuid